web/src/lib/components/Auction.svelte

```diff
@@ -92,6 +93,7 @@
                 <div class="lg:w-1/3">
                     <h2 class="text-3xl text-center mb-4">{auction.title}</h2>
                     <Gallery photos={auction.media} />
```

Comment:
I would much rather prefer we show/hide the components where they are used rather than having the component know it should show only in some cases. That makes them more reusable.

Reply:
Agreed, that is much better, to keep things modular.

Reply:
Fixed in [a0fbd21](https://github.com/PlebeianTech/plebeian-market/pull/12/commits/a0fbd21dbd8cb4798388020e7d391be0603ae620)